composer.json

```
@@ -26,7 +26,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ^8.0",
```

Please change the version constrain here because no one knows what BC breaks the new version will bring:
```suggestion
        "php": "^7.3 || ~8.0.0 || ~8.1.0",
```